### DISCUSSION TITLE
Dowload/Export all local replays

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20841] [Ideas]
Allowing players to download all their local replays with a single button, either for archiving, or for showing all their scores, would be a nice feature.
The export would be put in a zip with each map getting its own folder.
maybe notify the user of scores that are on local only maps (modified maps) so they can export that mapset for later.

**Comment by FibreTTP** [comment]
If this ever gets looked at again, I suggest that the replay files aren't exported zipped. I imagine a system where I could simply press export all, then an external program like Syncthing can synchronise it to a server. It'd help if osu! detects existing replays in the destination and does not overwrite (just skips).
